README.md

```diff
@@ -417,6 +417,24 @@ def run(nre):
     bob.send(...)
 ```
 
+## Short string literals
+
+From [cairo-lang docs](https://www.cairo-lang.org/docs/how_cairo_works/consts.html#short-string-literals): A short string is a string whose length is at most 31 characters, and therefore can fit into a single field element.
+
+In Nile, arguments to contract calls (calldata) that are neither int nor hex, are treated as short strings and converted automatically to the corresponding felt representation. Because of this, you can run the next from the CLI:
+
+```sh
+nile deploy MyToken 'MyToken name' 'MyToken symbol' (...)
+```
+
+And this is equivalent to passing the felt representation directly like this:
+
+```sh
+nile deploy MyToken 0x4d79546f6b656e206e616d65 0x4d79546f6b656e2073796d626f6c (...)
+```
+
+Note that if you want to pass the token name as a hex or an int, you need to provide the felt representation directly because these values are not interpreted as short strings.
```

Comment:
Do you think it'd be helpful to give a quick example on how to manually convert? Maybe just a link to `str_to_felt`?

Reply:
Or perhaps this should be expected considering that's how it was prior. Thoughts?

Reply:
I think adding the how-to is helpful in any case, so I added an entry for this. Let me know what you think, and any updates to the text (my English is far from good) is appreciated.